docs/src/app/components/pages/components/RadioButton/ExampleSimple.jsx

```diff
@@ -28,6 +31,14 @@ const RadioButtonExampleSimple = () => (
         style={styles.radioButton}
       />
     </RadioButtonGroup>
+
+    <RadioButtonGroup name="notRight" labelPosition="left" style={styles.block}>
```

Comment:
> Because labelPosition is a RadioButtonGroup property

Oh, I was expecting to see the `labelPosition="left"` on the `RadioButton` component instead.